content/en/blog/_posts/2022-12-01-cpumanager-goes-GA.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: blog
+title: 'Kubernetes 1.26: CPUManager goes GA'
+date: 2022-12-01
+slug: cpumanager-goes-GA
+---
+
+**Author:**
+Francesco Romani (Red Hat)
+
+The CPU Manager is a part of the kubelet, the Kubernetes node agent, which enables the user to allocate exclusive CPUs to containers.
+Since Kubernetes 1.10, where it [graduated to Beta](blog/2018/07/24/feature-highlight-cpu-manager/), the CPU Manager proved itself reliable and
```

Comment:
```suggestion
Since Kubernetes v1.10, where it [graduated to Beta](blog/2018/07/24/feature-highlight-cpu-manager/), the CPU Manager proved itself reliable and
```

Reply:
ditto :)